version.txt

```diff
@@ -43,6 +44,7 @@ harvard_forest.json,2.0.0
 home_ranges.json,1.1.2
 intertidal_abund_me.py,1.5.3
 iris.json,1.3.1
+json_test.json,1.2.1
```

Comment:
This script should start from 1.0.0. And since it is derevide from portal, and it is used for tests, we can move it to the `test/raw_data/scripts/`  directory

Reply:
Done